.github/workflows/testing.yml

```diff
@@ -36,10 +36,11 @@ jobs:
           restore-keys: |
             zig-
 
-      - run: |
+      - name: Prepare to publish
+        run: |
           rm yarn.lock
-          git grep -lz '@solarwinds-apm/' | xargs -0 sed -ie 's:@solarwinds-apm/:@${{ github.repository_owner }}/solarwinds-apm-:g'
-          git grep -lz -G 'solarwinds-apm[^-]' | xargs -0 sed -rie 's:solarwinds-apm([^-]):@${{ github.repository_owner }}/solarwinds-apm\1:g'
+          git grep -Ilz '@solarwinds-apm/' | xargs -0 sed -ie 's:@solarwinds-apm/:@${{ github.repository_owner }}/solarwinds-apm-:g'
+          git grep -Ilz -G 'solarwinds-apm[^-]' | xargs -0 sed -rie 's:solarwinds-apm([^-]):@${{ github.repository_owner }}/solarwinds-apm\1:g'
```

Comment:
The `I` flag excludes binary files from the results, previously this would rewrite strings inside the liboboe binaries (only happens on Linux) which completely breaks them